browser/ui/webui/brave_rewards_page_ui.cc

```diff
@@ -149,6 +149,12 @@ class RewardsDOMHandler : public WebUIMessageHandler,
   void GetExternalWallet(const base::ListValue* args);
   void OnGetExternalWallet(const ledger::type::Result result,
                            ledger::type::ExternalWalletPtr wallet);
+  void OnExternalWalletSelected(const ledger::type::Result result,
+                                ledger::type::ExternalWalletPtr wallet);
+
+  base::Value ParseExternalWalletPtr(const ledger::type::Result result,
```

Comment:
Nit: We're not parsing data here, so much as converting to a `Value` - `ExternalWalletToValue` or `ExternalWalletToJSON` might be a more appropriate name.

Also nit: since we're just reading the `ExternalWallet` data, we could just pass a `const ledger::type::ExternalWallet&`.

Reply:
resolved